views/layout.hbs

```diff
@@ -10,7 +10,7 @@
   </head>
 
   <body>
-
+    <a href="/drones" class="drones"> <span class="drones-word">Drones</span></a>
     {{{body}}}
 
```

Comment:
Just one piece of advice, add buttons to all the pages when doing the lab for a better navigation experience! :)